engine/docker-overview.md

```diff
@@ -267,8 +267,8 @@ the future, Docker may support other container formats by integrating with
 technologies such as BSD Jails or Solaris Zones.
 
 ## Next steps
-- Read about [Installing Docker Engine](installation/index.md#installation).
-- Get hands-on experience with the [Get Started With Docker](getstarted/index.md)
+- Read about [Install Docker Engine](installation/index.md#installation).
```

Comment:
Thank you. I restored it to “Installing". What about the link text in the following item? Should it also be "Getting started with Docker"?